# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxWindowsPlatform.h
# Commit: b4b4977857c7
# Full Hash: b4b4977857c7805b6e8c3599ce2e0b34badb655a
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2023-02-07 04:35:34
# Regressor Bug: 1799258
# File Overlap Count: 1
# Description:
#   Bug 1799258 - Do color-management on Windows+DComp via IDCompositionFilterEffects. r=sotaro
#   
#   + Add gfx.color_management.rec709_gamma_as_srgb:true. :'(
#   
#   In particular, rec709(16/255) -> srgb(31/255). Even though it's
# ==============================================================================

diff -r 2c23929f52f2 -r b4b4977857c7 gfx/thebes/gfxWindowsPlatform.h
--- a/gfx/thebes/gfxWindowsPlatform.h	Mon Feb 06 19:58:53 2023 +0000
+++ b/gfx/thebes/gfxWindowsPlatform.h	Mon Feb 06 19:58:54 2023 +0000
@@ -200,7 +200,13 @@
 
  protected:
   bool AccelerateLayersByDefault() override { return true; }
+
   nsTArray<uint8_t> GetPlatformCMSOutputProfileData() override;
+
+ public:
+  static nsTArray<uint8_t> GetPlatformCMSOutputProfileData_Impl();
+
+ protected:
   void GetPlatformDisplayInfo(mozilla::widget::InfoObject& aObj) override;
 
   void ImportGPUDeviceData(const mozilla::gfx::GPUDeviceData& aData) override;